patches/newDashboard/styles.css

```
@@ -41,8 +41,7 @@
         padding-bottom: 0;
     }
 
-    .tile.box:has(.lucky-number__circle.lucky-number__circle-1),
-    .tile.box:has(.tile__lucky-number) {
+    .tile.box.tile__lucky-number {

```

```suggestion
    .tile.box.tile__lucky-number,
    .tile.box:has(.lucky-number__circle.lucky-number__circle-1),
    .tile.box:has(.tile__lucky-number) {
```
one of selectors can still apply for days without lucky number turned on, also this change is applied with version currently published on mozilla addons